packages/web/utils/helper.js

```diff
@@ -293,3 +293,22 @@ export const getYoutubeVideoId = (link) => {
 	const match = link.match(regex);
 	return match && match[7].length === 11 ? match[7] : null;
 };
+
+/**
+ * Handle phone number mask, necessary to alternate between masks when phone has 9 digits
+ *
+ * @returns {object}
+ */
+export const beforeMaskedValueChange = (newState) => {
```

Comment:
Very nice idea to get this into a helper file!

I also think it would be better to replace that `beforeMaskedValueChange` at `contact-us` page by this one, it will help us keep the code DRYer :smile: